setup.cfg

```diff
@@ -39,6 +39,7 @@ install_requires =
     setuptools>=75.8.1
     tqdm~=4.67.0
     types-PyYAML~=6.0
+    yamale>=6.0.0
```

Comment:
To avoid the [dependency confusion vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), there should probably be an upper limit on `yamale`, or just use `yamale~=6.0.0` 

(Same would apply to `setuptools`, a few lines up)

Reply:
Woot woot! 🎉